core/src/main/java/greencity/controller/UserController.java

```diff
@@ -59,20 +64,26 @@ public class UserController {
      */
     @Operation(summary = "Update status of user")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = HttpStatuses.OK,
-            content = @Content(schema = @Schema(implementation = UserStatus.class))),
-        @ApiResponse(responseCode = "303", description = HttpStatuses.SEE_OTHER),
-        @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
-        @ApiResponse(responseCode = "403", description = HttpStatuses.FORBIDDEN)
+            @ApiResponse(responseCode = "200", description = HttpStatuses.OK,
+                    content = @Content(schema = @Schema(implementation = UserStatus.class))),
+            @ApiResponse(responseCode = "303", description = HttpStatuses.SEE_OTHER),
+            @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
+            @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
+            @ApiResponse(responseCode = "403", description = HttpStatuses.FORBIDDEN)
     })
     @PatchMapping("status")
     public ResponseEntity<UserStatusDto> updateStatus(
-        @Valid @RequestBody UserStatusDto userStatusDto, @ApiIgnore Principal principal) {
-        return ResponseEntity.status(HttpStatus.OK)
-            .body(
-                userService.updateStatus(
-                    userStatusDto.getId(), userStatusDto.getUserStatus(), principal.getName()));
+            @Valid @RequestBody UserStatusDto userStatusDto, @ApiIgnore Principal principal) {
+        if (principal == null) {
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
```

Comment:
In my opinion, this block if(principal == null) doesn't make any sense. In SecuritiConfig configured that only ADMIN can reach this endpoint. It means the principal can't be null!

Also, task 314 says "Sign-in us user". According to SecurityConfig this endpoint can't reach anybody except ADMIN.

I was trying to sign in as USER and enter valid data. And I received 401. But expected 403.
Please fix it.